### ISSUE TITLE
Feature Request: Allow Running Parity Outside of Docker

### ISSUE THREAD

**Issue by gaia** [ambrosus/ambrosus-nop#114]
my node (the docker host) runs with a LAN address. Node is working fine but I see the firewall dropping a lot of connections on 30303.

I would like to create a NIC dedicated for the parity docker, so that the docker container uses an interface managed by the firewall (10.10.10.x), not a 172.0.20.x.

Any suggestions on how to go about that? The options I've looked into are

- macvlan https://docs.docker.com/network/macvlan/
- pipework https://github.com/jpetazzo/pipework#connect-a-container-to-a-local-physical-interface
- create a docker bridge to the dedicated docker NIC https://docs.docker.com/v17.09/engine/userguide/networking/default_network/custom-docker0/
- bind ports on the host (docker keeps it's own LAN) https://docs.docker.com/v17.09/engine/userguide/networking/default_network/binding/
There is no need to reinvent the wheel. Is anyone using a solution for this (from the above options or not)?

Long story short: **It'd be much simpler for me to use parity outside of docker. Is this possible?**
